src/Reflection/ClassReflection.php

```diff
@@ -590,6 +613,11 @@ public function getProperty(string $propertyName, ClassMemberAccessAnswerer $sco
 				}
 				$this->properties[$key] = $property;
 			}
+
+			if ($this->requireExtendsPropertiesClassReflectionExtension->hasProperty($this, $propertyName)) {
+				$property = $this->requireExtendsPropertiesClassReflectionExtension->getProperty($this, $propertyName);
+				$this->properties[$key] = $property;
```

Comment:
Why are you overriding a property that might have already been set in the foreach above?